packages/google_maps_flutter/google_maps_flutter/ios/Classes/google_maps_flutter-umbrella.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import <google_maps_flutter/FLTGoogleMapsPlugin.h>
```

Comment:
From CI:

```
Testing failed:
	Umbrella header for module 'google_maps_flutter' does not include header 'JsonConversions.h'
	Umbrella header for module 'google_maps_flutter' does not include header 'FLTGoogleMapTileOverlayController.h'
```

(JsonConversions.h shouldn't actually be public, but you don't need to fix that in this PR; it's a pre-existing mistake. @jmagman Maybe we should do an audit of the headers our plugins export, and make a repo convention for paths to put headers that aren't intended to be public?)

Reply:
I added `FLTGoogleMapTileOverlayController.h`.

Reply:
Still getting 
```
 - WARN  | [iOS] xcodebuild:  Headers/Private/google_maps_flutter/google_maps_flutter-umbrella.h:10:1: warning: umbrella header for module 'google_maps_flutter' does not include header 'JsonConversions.h' [-Wincomplete-umbrella]
 - ```

Reply:
To clarify my "JsonConversions.h shouldn't actually be public, but you don't need to fix that in this PR" comment: you don't need to fix the fact that it's public and shouldn't be in this PR. You do need to fix the fact that as a (currently) public header, it needs to be in the umbrella.

Reply:
Ah wait, do I want these 2 headers in `FLTGoogleMapsPlugin.h` or in the umbrella? Right now they're in the plugin header...

Reply:
I moved them to the umbrella